interchain.go

```diff
@@ -226,7 +230,17 @@ func (ic *Interchain) Build(ctx context.Context, rep *testreporter.RelayerExecRe
 			)
 		}
 
-		if err := rp.Relayer.LinkPath(ctx, rep, rp.Path); err != nil {
+		// If channel creation options are not fully specified, default to the ics20 fungible token transfer channel options.
+		if !opts.CreateChannelOpts.IsFullyConfigured() {
+			opts.CreateChannelOpts = ibc.DefaultChannelOpts()
```

Comment:
Should we return an err here? Ideally telling the user what data they are missing?

Reply:
If we want to change `IsFullyConfigured() bool` to `Validate() error` then this would be ideal. 
Then we can either receive a fully configured `CreateChannelOpts` object with valid values or we return an error instead of trying to default to some acceptable behavior.

Reply:
I'll approve for the first pass. But it's probably a better user experience to know what data they're missing.

Reply:
To be clear, if a detailed error isn't in first pass that's fine - however, I think we need to return an error here or else the user may get unexpected behavior that's tricky to debug. If I expect a certain channel version and see it's defaulting to another, I'd be quite confused.

Reply:
I agree. I'm just going to include those changes real quick as well, thanks!